community/index.tt

```diff
@@ -300,6 +300,10 @@
       <p>Maintains and releases the Nix package manager.</p>
       <a class="button -primary" href="[%root%]community/teams/nix.html">Read more</a>
 
+      <h3>CUDA Team</h3>
+      <p>Responsible for all things CUDA-related in Nixpkgs.</p>
```

Comment:
This is only my personal opinion, so feel free to ignore, since you're running that show: It's a bit too colloquial for my taste.

```suggestion
      <p>Maintains CUDA-related software in Nixpkgs.</p>
```